Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -1463,7 +1463,7 @@ void MainWindow::UpdateScreenSaverInhibition()
 
   m_is_screensaver_inhibited = inhibit;
 
-#if defined(HAVE_XRANDR) && HAVE_XRANDR
+#if defined(HAVE_X11) && HAVE_X11
```

Comment:
Could you simplify this to `#ifdef HAVE_X11`? We never define it to 0.

We should probably do the same thing for HAVE_XRANDR after removing this line from the CMakeLists.txt: https://github.com/dolphin-emu/dolphin/blob/ddacbf83f65b9642f3c9cee9e075b7ec3d473321/CMakeLists.txt#L444

Reply:
Ok, did add another commit cleaning up the HAVE_XRANDR guards